tests/examples/wallet/test_wallet.py

```diff
@@ -1,30 +1,34 @@
-from ethereum.tools import tester as t
-from ethereum import utils
-from vyper import compiler
+import pytest
 
 
-t.languages['vyper'] = compiler.Compiler()
-t.s = t.Chain()
-t.s.head_state.gas_limit = 10**9
-
-x = t.s.contract(open('examples/wallet/wallet.v.py').read(), args=[[t.a1, t.a2, t.a3, t.a4, t.a5], 3], language='vyper')
-print(t.s.last_tx.data[-192:])
-# Sends wei to the contract for future transactions gas costs
-t.s.tx(sender=t.k1, to=x.address, value=10**17)
-
-print([utils.encode_hex(a) for a in [t.a1, t.a2, t.a3, t.a4, t.a5]])
-
+@pytest.fixture
+def c(w3, tester, get_contract):
+    a0, a1, a2, a3, a4, a5, a6 = w3.eth.accounts[:7]
+    with open('examples/wallet/wallet.v.py') as f:
+        code = f.read()
+    # Sends wei to the contract for future transactions gas costs
+    import ipdb; ipdb.set_trace()
```

Comment:
I used to use `ipdb` but I've recently switched to `pdbpp` which allows you to just use the built-in `pdb` but to get a nice repl.  It also works with `pytest --pdb` natively.